run

```diff
@@ -117,6 +120,15 @@ while [[ -n "${1:-}" ]] && [[ "${1:0:1}" == "-" ]]; do
     shift
 done
 
+kill_docker_container() {
+  container_name="${1}"
+  # Kill any previously running containers
+  if docker ps | grep -q ${container_name}; then
+    echo "Killing previous container:"
+    docker kill ${container_name} || true
```

Comment:
Although `--rm` shouldn't leave stopped containers behind, if a stopped container does get left behind then the "run" command will fail. We should delete stopped containers as well, by using `docker ps --all` for detection, and `docker rm --force` for killing & removing.